src/component/AboutPage/Leadership/LeadershipComponenents/LeadershipCard/leadershipCard.jsx

```diff
@@ -1,9 +1,6 @@
 import React, { useState } from "react";
 import ReactCardFlip from "react-card-flip";
-import Col from "react-bootstrap/Col";
-import Image from "react-bootstrap/Image";
-import Row from "react-bootstrap/Row";
-import ShadowedButton from "../../ShadowedButton";
+import ShadowedButton from "../../../../ShadowedButton";
```

Comment:
Is your prettier set up? It should remove "" to ''

Reply:
I have prettier installed, but I don't think it's working properly or maybe I don't have it showing the errors because I don't get problems.